docs/wiki/[Examples]-Deploy-Custom-Management-Landing-Zone.md

```diff
@@ -0,0 +1,131 @@
+
+```hcl
+    settings = {
+      log_analytics = {
+        enabled = false
+     }
+```
+If you've already deployed management resources, this will allow you to enable or disable specific parameters.
+Every parameter set to true will create new principal id's that will force replacement in place of the existing
+roles and policies to reassign them to the log analytics and sentinel resources specified. Likewise, if set to false, existing configurations will be removed.  
+
+If location is not specified, the resources will default to *eastus*
+```hcl
+    terraform {
+      required_providers {
+        azurerm = {
+            source = "hashicorp/azurerm"
+            version = ">=2.46.1"
+        }
+   
+     }
+    }
+
+    provider "azurerm"{
+    features {}
+    }
+
+    #Pull current Tenant ID from connection settings and store to data source
+    
+    data "azurerm_client_config" current {}
+
+    module "enterprise_scale" {
+      source = "Azure/caf-enterprise-scale/azurerm"
+      version = "0.3.1"
+
+    root_parent_id = data.azurerm_client_config.current.tenant_id
+    root_id = "contoso" 
+    root_name = "Contoso"
+    deploy_management_resources = "true"
+    subscription_id_management = "<insert_subscription_id>" 
+    configure_management_resources = {
+        settings = {
+      log_analytics = {
+        enabled = true
+        config = {
+          retention_in_days                           = 30
+          enable_monitoring_for_arc                   = true
+          enable_monitoring_for_vm                    = true
+          enable_monitoring_for_vmss                  = true
+          enable_solution_for_agent_health_assessment = true
+          enable_solution_for_anti_malware            = false
+          enable_solution_for_azure_activity          = true
+          enable_solution_for_change_tracking         = true
+          enable_solution_for_service_map             = false
+          enable_solution_for_sql_assessment          = true
+          enable_solution_for_updates                 = true
+          enable_solution_for_vm_insights             = true
+          enable_sentinel                             = true
+        }
+      }
+      security_center = {
+        enabled = true
+        config = {
+          email_security_contact             = "email@contoso.com"
+          enable_defender_for_acr            = true
+          enable_defender_for_app_services   = true
+          enable_defender_for_arm            = true
+          enable_defender_for_dns            = true
+          enable_defender_for_key_vault      = true
+          enable_defender_for_kubernetes     = false
+          enable_defender_for_servers        = true
+          enable_defender_for_sql_servers    = true
+          enable_defender_for_sql_server_vms = true
+          enable_defender_for_storage        = true
+        }
+      }
+        }
+    location = "centralus"
```

Comment:
For consistency, we should probably use `location = "eastus"` for these (or maybe leave as the default `location = null`), unless trying to explicitly highlight a change of region for these resource.